From da233d204ee5abc76d8e0a152d477f8e161a15f4 Mon Sep 17 00:00:00 2001
From: "Eric S. Raymond" <esr@thyrsus.com>
Date: Fri, 9 Dec 2016 19:59:25 -0500
Subject: [PATCH 254/268] Address GitLab issue #200: compiler warnings...

...when building with glibc 2.10.

This code was pretty obviously wrong, not clear how it wasn't spotted sooner.
---
 ntptime/ntptime.c | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/ntptime/ntptime.c b/ntptime/ntptime.c
index fcff7aa..9689684 100644
--- a/ntptime/ntptime.c
+++ b/ntptime/ntptime.c
@@ -50,11 +50,7 @@
 
 /* MUSL port shim */
 #ifndef HAVE_NTP_GETTIME
-#ifdef STRUCT_NTPTIMEVAL_HAS_TAI
 int ntp_gettime(struct ntptimeval *ntv)
-#else
-int ntp_gettime(struct timex *ntv)
-#endif
 {
 	struct timex tntx;
 	int result;
@@ -121,11 +117,7 @@ main(
 {
 	extern int ntp_optind;
 	extern char *ntp_optarg;
-#ifdef STRUCT_NTPTIMEVAL_HAS_TAI
 	struct ntptimeval ntv;
-#else
-	struct timex ntv;
-#endif
 	struct timeval tv;
 	struct timex ntx, _ntx;
 	int	times[20];
-- 
2.7.4

